commit 18fb5623cf2edf70ab8eadab1b2a85251fbd9ee0
author: dude
date: 2017-06-30T13:47:30+03:00
revision: 2969517

    Add access to some hidden-in-implementation methods
    ISSUE: LIB-693
    REVIEW: 293505

--- a/include/cctz/time_zone.h
+++ b/include/cctz/time_zone.h
@@ -307,2 +310,6 @@
 
+// Access to convert functions which are placed in time_zone_if.h
+time_point<seconds> UnixSecondsToTimePoint(std::int_fast64_t);
+std::int_fast64_t TimePointToUnixSeconds(const time_point<seconds>& tp);
+
 }  // namespace cctz
--- a/src/time_zone_if.cc
+++ b/src/time_zone_if.cc
@@ -33,2 +33,10 @@
 
+time_point<seconds> UnixSecondsToTimePoint(std::int_fast64_t seconds) {
+  return FromUnixSeconds(seconds);
+}
+
+std::int_fast64_t TimePointToUnixSeconds(const time_point<seconds>& tp) {
+  return ToUnixSeconds(tp);
+}
+
 }  // namespace cctz
